apps/meteor/app/ui-message/client/messageBox/messageParser.ts

```
@@ -0,0 +1,131 @@
+						}
+
+						// Changing the shortcode to display rendition of the custom uploaded
+						if (emoji.image) {
+							const rawUrl = emoji.image.replace(/^url\(["']?/, '').replace(/["']?\)$/, '');
+							return (
+								`<span style="font-size:0.1px">` +
+								`<img src="${rawUrl}" style="width:1.5rem;height:1.5rem;vertical-align:middle"/>` +
+								`:${token.shortCode}:</span>`
+							);
+						}
+					} else return escapeHTML(token.unicode || '');
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)